pages/feed/index.tsx

```diff
@@ -159,6 +161,30 @@ const Feeds = () => {
               </div>
             </div>
           )}
+          {meta.pageCount > 1 && (
+            <div className="mt-10 max-w-[48rem] flex px-2 items-center justify-between">
+              <div>
+                <PaginationResults
+                  from={meta.page === 1 ? (meta.itemCount > 0 ? meta.page : 0) : (meta.page - 1) * meta.limit + 1}
+                  to={meta.page * meta.limit <= meta.itemCount ? meta.page * meta.limit : meta.itemCount}
```

Comment:
I've noticed this pattern being repeated in both this PR and in the pre-existing implementations of the component. Maybe this logic should be done inside the component to avoid code repetition?

Reply:
I agree with your suggestions. Nice catch! 👍